models/tokenlon/ethereum/tokenlon_v5_ethereum_amm_v1_trades.sql

```diff
@@ -95,23 +104,23 @@ LEFT JOIN {{ ref('tokens_erc20') }} erc20a
 LEFT JOIN {{ ref('tokens_erc20') }} erc20b 
     ON erc20b.contract_address = dexs.token_sold_address
     AND erc20b.blockchain = 'ethereum'
-LEFT JOIN {{ source('prices', 'usd') }} p_bought 
-    ON p_bought.minute = date_trunc('minute', dexs.block_time)
+LEFT JOIN prices p_bought
+    ON p_bought.hour = date_trunc('hour', dexs.block_time)
```

Comment:
Do you have any specific reason to change `minute` into `hour`?
Like increasing precision or stability?

Reply:
I found that using `minute`, there is missing data for certain minutes in the prices table. I can only take the average values at the `hour`

Reply:
fyi @Hosuke -- this workaround relates to my latest comment in the dex issue for assigning `amount_usd`. maybe aggregating to hourly is another idea to use universally 🤔 can add to brainstorm list of ideas

Reply:
We may have avg-hour prices table as an independent spell.

Reply:
let's approve this PR as-is in the old lineage, then continue conversation on approach in issue